src/replicache.ts

```diff
@@ -176,6 +193,7 @@ export class Replicache<MD extends MutatorDefs = {}> {
 
   /** The schema version of the data understood by this application. */
   schemaVersion: string;
+  previousSchemaVersions: string[];
```

Comment:
Why public read/write property? All public API should have JSDoc

Also, we should make schemaVersion readonly. Changing it at runtime does not seem meaningful

Reply:
yeah the perviousSchemaVersions is not fully worked out. I think I'm going to take your advice and do this internally so it doesnt have to be part of the public api.

should name also be readonly?